Merge Conflict/Assets/Scripts/AudioManager/AudioMixerLevels.cs

```diff
@@ -0,0 +1,63 @@
+/**********************************************************************************************************************
+Name:          AudioMixerLevels
+Description:   Handles the volume level of the slider in the audio settings and saves the values in the playerprefs.  
+Author(s):     Daniel Rittrich
+Date:          2024-03-24
+Version:       V1.1
+TODO:          - /
+**********************************************************************************************************************/
+
+using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.Audio;
+
+public class AudioMixerLevels : MonoBehaviour
+{
+    public Slider thisSlider;
+    public AudioMixer audioMixer;
+
+    [Tooltip("Must be 'volumeEffects' or 'volumeMusic' !")]
```

Comment:
refactor exposedParameter and playerPrefsKey to be driven by an enum. 
In places where these variables are used, should be a function call, that takes in the enum as parameter and returns the approriate string ("volumeEffects", ...)

Reason is that it is to easy to accidentily change these values in the editor and not notice the error.